src/drag-drop/dragAndDrop.js

```diff
@@ -62,6 +62,22 @@ angular.module('org.bonitasoft.dragAndDrop',[])
       });
     };
 
+    /**
+     * Try to find the node that initiate the dragEvent
+     * @param  {Node} node the event.target node
+     * @return {Node}      the found node or null
+     */
+    this.getDragInitiatorNode = function getDragInitiatorNode(node) {
+      var currentNode = node;
+      while(currentNode.parentNode) {
+        if (currentNode.getAttribute('draggable') === 'true') {
```

Comment:
no and it should not
to make and element draggrable, you need to set a draggable=true attribute
https://developer.mozilla.org/en-US/docs/Web/Guide/HTML/Drag_operations#dragstart